meka/srcs/z80marat/Tables.h

```diff
@@ -374,42 +374,42 @@ static byte CyclesED[256] =
 
 static byte CyclesXX[256] =
 {
-   0, 0, 0, 0, 0, 0, 0, 0, 0,15, 0, 0, 0, 0, 0, 0,
-   0, 0, 0, 0, 0, 0, 0, 0, 0,15, 0, 0, 0, 0, 0, 0,
-   0,14,20,10, 9, 9, 9, 0, 0,15,20,10, 9, 9, 9, 0,
-   0, 0, 0, 0,23,23,19, 0, 0,15, 0, 0, 0, 0, 0, 0,
-   0, 0, 0, 0, 9, 9,19, 0, 0, 0, 0, 0, 9, 9,19, 0,
-   0, 0, 0, 0, 9, 9,19, 0, 0, 0, 0, 0, 9, 9,19, 0,
-   9, 9, 9, 9, 9, 9,19, 9, 9, 9, 9, 9, 9, 9,19, 9,
-  19,19,19,19,19,19,19,19, 0, 0, 0, 0, 9, 9,19, 0,
-   0, 0, 0, 0, 9, 9,19, 0, 0, 0, 0, 0, 9, 9,19, 0,
-   0, 0, 0, 0, 9, 9,19, 0, 0, 0, 0, 0, 9, 9,19, 0,
-   0, 0, 0, 0, 9, 9,19, 0, 0, 0, 0, 0, 9, 9,19, 0,
-   0, 0, 0, 0, 9, 9,19, 0, 0, 0, 0, 0, 9, 9,19, 0,
-   0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
-   0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
-   0,14, 0,23, 0,15, 0, 0, 0, 8, 0, 0, 0, 0, 0, 0,
-   0, 0, 0, 0, 0, 0, 0, 0, 0,10, 0, 0, 0, 0, 0, 0
+   8,14,11,10, 8, 8,11, 8, 8,15,11,10, 8, 8,11, 8,
+  12,14,11,10, 8, 8,11, 8,16,15,11,10, 8, 8,11, 8,
+  11,14,20,10, 8, 8,11, 8,11,15,20,10, 8, 8,11, 8,
+  11,14,17,10,23,23,19, 8,11,15,17,10, 8, 8,11, 8,
+   8, 8, 8, 8, 8, 8,19, 8, 8, 8, 8, 8, 8, 8,19, 8,
+   8, 8, 8, 8, 8, 8,19, 8, 8, 8, 8, 8, 8, 8,19, 8,
+   8, 8, 8, 8, 8, 8,19, 8, 8, 8, 8, 8, 8, 8,19, 8,
+  19,19,19,19,19,19, 8,19, 8, 8, 8, 8, 8, 8,19, 8,
+   8, 8, 8, 8, 8, 8,19, 8, 8, 8, 8, 8, 8, 8,19, 8,
+   8, 8, 8, 8, 8, 8,19, 8, 8, 8, 8, 8, 8, 8,19, 8,
+   8, 8, 8, 8, 8, 8,19, 8, 8, 8, 8, 8, 8, 8,19, 8,
+   8, 8, 8, 8, 8, 8,19, 8, 8, 8, 8, 8, 8, 8,19, 8,
+   9,14,14,14,14,15,11,15, 9,14,14, 0,14,21,11,15,
+   9,14,14,15,14,15,11,15, 9, 8,14,15,14, 4,11,15,
+   9,14,14,23,14,15,11,15, 9, 8,14, 8,14, 4,11,15,
+   9,14,14, 8,14,15,11,15, 9,10,14, 8,14, 4,11,15

```

Comment:
For DD/FD instructions. This looks alright.
DD and FD have 0 in Cycles[] tables and there's no hidden extra in code.